src/Compilers/VisualBasic/Portable/BoundTree/Expression.vb

```diff
@@ -602,7 +671,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             End Get
         End Property
 
-        Private ReadOnly Property IBinaryKind As BinaryOperationKind Implements IBinaryOperatorExpression.BinaryKind
```

Comment:
It's a naming convention used through out IOperation, I didn't change any of it.